{
   "active_lock_reason" : null,
   "assignee" : null,
   "assignees" : [],
   "author_association" : "MEMBER",
   "body" : "Currently, trying to load a descriptor (sqlite) wallet that is already loaded throws the following error:\r\n> error code: -4\r\n> error message:\r\n> Wallet file verification failed. SQLiteDatabase: Unable to obtain an exclusive lock on the database, is it being used by another instance of Bitcoin Core?\r\n\r\nI don't think it is very clear what it means for a user.\r\n\r\nWhile a legacy wallet would throw:\r\n> error code: -35\r\n> error message:\r\n> Wallet file verification failed. Refusing to load database. Data file '/home/user/.bitcoin/signet/wallets/test_wallet/wallet.dat' is already loaded.\r\n\r\nThis PR changes the error message for both types of wallet to:\r\n> error code: -35\r\n> error message:\r\n> Wallet file verification failed. Wallet \"test_wallet\" is already loaded.",
   "closed_at" : null,
   "closed_by" : null,
   "comments" : 0,
   "comments_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/26192/comments",
   "created_at" : "2022-09-28T13:15:37Z",
   "draft" : false,
   "events_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/26192/events",
   "html_url" : "https://github.com/bitcoin/bitcoin/pull/26192",
   "id" : 1389342976,
   "labels" : [],
   "labels_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/26192/labels{/name}",
   "locked" : false,
   "milestone" : null,
   "node_id" : "PR_kwDOABII584_xx4j",
   "number" : 26192,
   "performed_via_github_app" : null,
   "pull_request" : {
      "diff_url" : "https://github.com/bitcoin/bitcoin/pull/26192.diff",
      "html_url" : "https://github.com/bitcoin/bitcoin/pull/26192",
      "merged_at" : null,
      "patch_url" : "https://github.com/bitcoin/bitcoin/pull/26192.patch",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/pulls/26192"
   },
   "reactions" : {
      "+1" : 0,
      "-1" : 0,
      "confused" : 0,
      "eyes" : 0,
      "heart" : 0,
      "hooray" : 0,
      "laugh" : 0,
      "rocket" : 0,
      "total_count" : 0,
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/26192/reactions"
   },
   "repository_url" : "https://api.github.com/repos/bitcoin/bitcoin",
   "state" : "open",
   "state_reason" : null,
   "timeline_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/26192/timeline",
   "title" : "rpc: Improve error when wallet is already loaded",
   "updated_at" : "2022-09-28T13:15:37Z",
   "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/26192",
   "user" : {
      "avatar_url" : "https://avatars.githubusercontent.com/u/22493292?v=4",
      "events_url" : "https://api.github.com/users/aureleoules/events{/privacy}",
      "followers_url" : "https://api.github.com/users/aureleoules/followers",
      "following_url" : "https://api.github.com/users/aureleoules/following{/other_user}",
      "gists_url" : "https://api.github.com/users/aureleoules/gists{/gist_id}",
      "gravatar_id" : "",
      "html_url" : "https://github.com/aureleoules",
      "id" : 22493292,
      "login" : "aureleoules",
      "node_id" : "MDQ6VXNlcjIyNDkzMjky",
      "organizations_url" : "https://api.github.com/users/aureleoules/orgs",
      "received_events_url" : "https://api.github.com/users/aureleoules/received_events",
      "repos_url" : "https://api.github.com/users/aureleoules/repos",
      "site_admin" : false,
      "starred_url" : "https://api.github.com/users/aureleoules/starred{/owner}{/repo}",
      "subscriptions_url" : "https://api.github.com/users/aureleoules/subscriptions",
      "type" : "User",
      "url" : "https://api.github.com/users/aureleoules"
   }
}
